.github/workflows/jit.yml

```
@@ -128,7 +128,7 @@ jobs:
           choco install llvm --allow-downgrade --no-progress --version ${{ matrix.llvm }}.1.0
           ./PCbuild/build.bat --experimental-jit ${{ matrix.debug && '-d' || '' }} -p ${{ matrix.architecture }}
 
-      - name: Native macOS
+      - name: macOS
```

Let's keep the old name, because there's also "Native Linux" and so on in this file:
```suggestion
      - name: Native macOS
```